resume/routes.py

```diff
@@ -14,49 +14,49 @@
 
 @app.route("/")
 def hello():
-    return render_template("home.html")
+    return render_template("home.html"),200
 
 
 
 
 @app.route("/register",methods=['GET','POST'])
 def register():
     if current_user.is_authenticated:
-        return redirect(url_for('hello'))
+        return redirect(url_for('hello')),200
     form = Reg()
     if form.validate_on_submit():
         hashed = bcrypt.generate_password_hash(form.password.data).decode('utf-8')
         new_user = user(username=form.username.data,email=form.email.data,password=hashed)
         db.session.add(new_user)
         db.session.commit()
         flash(f'Account Created for {form.username.data}!','success')
-        return redirect(url_for('login'))
+        return redirect(url_for('login')),201
     return render_template('register.html',title='Register',form=form)
 
 @app.route("/login",methods=['GET','POST'])
 def login():
     if current_user.is_authenticated:
-        return redirect(url_for('hello'))
+        return redirect(url_for('hello')),200
     form = Login()
     if form.validate_on_submit():
         logged = user.query.filter_by(email=form.email.data).first()
         if logged and bcrypt.check_password_hash(logged.password,form.password.data):
             login_user(logged,remember=form.remember.data)
             next_page = request.args.get('next')            
             if next_page:
-                return redirect(next_page)
+                return redirect(next_page),200
             else:
-                return redirect(url_for('hello'))
+                return redirect(url_for('hello')),200
         else:
-            flash('Login unsuccessful')
+            flash('Login unsuccessful'),401
```

Comment:
Hey it seems in flask it is preferred to have the syntax in this way as per the flask documentation

redirect(url_for('hello'),code=301) 

returning the status codes in the above method out of the redirect function  seems add bugs to the app as browser requests you to manually redirect to the page .

Also Flask supports status codes 301, 302, 303, 305, and 307 for redirect (as per the documentation you can check), you dont need to give for render_template the status codes . Could you please fix these for the routes.py code

Reply:
Alright I'll do it !

Reply:
Also please send the PR's to the development branch as the master is for kicking off heroku deploys , I merge all PRs into master after it is stable in development .